renovate.json

```diff
@@ -0,0 +1,147 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "local>elastic/renovate-config"
+  ],
+  "baseBranches": [
+    "main",
+    "9.7",
+    "9.6",
+    "9.5",
+    "9.4",
+    "9.3",
+    "9.2",
+    "9.1",
+    "8.19"
+  ],
```

Comment:
Should we maintain this list?
When 9.3 is out, we should remove 9.1?
https://www.elastic.co/support/eol

Reply:
IMO we should remove 9.1 after its last patch version released.
In our case `9.1.10` (expected Tue, Jan 06 2026).